src/Tab/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { HTMLAttributes, PureComponent, ReactNode } from 'react';
+import styled, { withTheme } from 'styled-components';
+
+import { gray600 } from '../Colors';
+import { caption1 } from '../TextStyles';
+import Theme from '../Theme';
+import { TabStyleProps } from './interface';
+import TabItem from './TabItem';
+import { getTabActiveColorByTheme, getTabBorderColorByTheme } from './utils';
+
+interface Tabs {
+  value: string | number;
+  title: string;
+  badge?: React.ReactNode;
+}
+
+interface TabProps extends TabStyleProps {
+  tabs: Tabs[];
+  currentValue: string | number;
+  fluid: boolean;
+  className?: string;
+  onTabChange?: (value: string | number) => void;
+  divAttributes?: HTMLAttributes<HTMLDivElement>;
+  children?: ReactNode;
+}
+
+class Tab extends PureComponent<TabProps> {
+  public static defaultProps = {
+    fluid: true,
+    type: 'default',
+    theme: Theme.light,
+  };
+
+  public render() {
+    const { className, tabs, type, theme, currentValue, onTabChange, fluid, divAttributes, children } = this.props;
+    return (
+      <div className={className} {...divAttributes}>
+        <TabList type={type} theme={theme}>
+          {tabs.map(tab => (
+            <TabItem
+              key={tab.value}
+              value={tab.value}
+              className={`${className ? `${className}-item ` : ''}${tab.value === currentValue ? 'active' : ''}`}
```

Comment:
active의 경우에는 prop으로 넘겨주고, TabItem내에서 styled-component로 스타일 분기를 처리하면 좋을 것 같습니다.
className은 string이기 때문에, 이 라인에서의 코드와 TabItem내의 스타일 코드에서 active라는 클래스 이름을 항상 맞춰줘야하기 때문에 실수가 일어나기 쉽습니당

`${className}-item` 는 어디에서 사용되나요?!

```suggestion
              active={tab.value === currentValue}
```

Reply:
요거는 조이의 코멘트를 참고해서 수정할 예정입니당

Reply:
다음 pr에서 수정되나욥?